src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -88,16 +88,11 @@ std::optional<std::array<double, Dim>> Translation<Dim>::inverse(
   double radial_function_value = 1.0;
   if (f_of_r_ != nullptr) {
     std::array<double, Dim> distance_to_center = target_coords;
-    double root_factor = 0.;
     for (size_t i = 0; i < Dim; i++) {
       // shifted center
       gsl::at(distance_to_center, i) -= gsl::at(center_, i);
-      // root_factor needed for root_finder
-      root_factor +=
-          -2.0 * gsl::at(distance_to_center, i) * function_of_time[i];
     }
-    const double root = root_finder(magnitude(distance_to_center),
-                                    sqrNorm(function_of_time), root_factor);
+    const double root = root_finder(distance_to_center, function_of_time);
```

Comment:
[Optional] No need for `distance_to_center` here. Just do

```
const double root = root_finder(target_coords - center_, function_of_time);
```

Reply:
Good point.  Will do.